css/index.css

```diff
@@ -93,36 +97,36 @@ header nav a:hover {
 }
 
 header .logo {
-    align-self: flex-end;
-    margin: 28px 0 0 91px;
+    margin-left: 8%;
 }
 
-/* Call To Action (CTA) Styles */
-
 .cta {
     display: flex;
-    margin-top: 72px;
-    justify-content: flex-end;
+    justify-content: space-between;
+    align-items: center;
+    margin: 40px 7% 50px;
 }
 
 .cta .cta-text {
-    font-size: 72px;
     text-align: center;
-    letter-spacing: 5px;
     display: flex;
     justify-content: center;
     flex-flow: column;
     align-items: center;
-    margin: 0 82px 0 0;
+}
+
+.cta .cta-text h1 {
+    font-size: 7.5vw;
```

Comment:
Impressive to see you using vw, continue going further than the knowledge that is given to you in lecture! 👍